src/q7/ml-ELEC2870/exam/2019/Janvier/All/ml-ELEC2870-exam-2019-Janvier-All.tex

```diff
@@ -0,0 +1,63 @@
+\documentclass[en]{../../../../../../eplexam}
+
+\hypertitle{Machine Learning}{7}{ELEC}{2870}{2019}{Janvier}{All}
+{ELEC students 2018-2019}
+{Michel Verleysen and John Lee}
+
+\section{Verleysen}
+
+\begin{itemize}
+    \item \textbf{Generalization (5)}: Explain how overfitting, generalization and complexity are linked together. Illustrate with some examples (in 2 or 3D) for both regression and classification.
+    \item \textbf{Feature selection}: Explain what feature extraction is. Why is it important to do feature extraction to obtain less but not too much features?
```

Comment:
```suggestion
    \item \textbf{Feature selection}: Explain what feature extraction is. Why is it important to do feature extraction to obtain less but not too many features?
```

"less but not too many"? Shouldn't it be "less but not too few"?

Reply:
I agree